thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -112,7 +112,7 @@
       {{#te-modal
         cancelButtonText="Cancel"
         submitButtonText="Report"
-        saveAction=(action "onSave")
+        submitAction=(action "onSave")
```

Comment:
fyi @newsummit had to make this change since I modified the component's API here https://github.com/linkedin/pinot/pull/2575/files#diff-3a3caf1f54b62eda5b463d1a8e9cc743R50